ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -152,6 +153,31 @@
         (get-unique-native-id context subscription))
       native-id)))
 
+(defn add-if-missing
+  [context subscription metadata user field value-fn]
+  (if-let [old-value (field metadata)]
+    subscription
+    (do
+      (println (value-fn context user))
```

Comment:
Swap this out with debug or remove the println

Reply:
This was left in by mistake - good catch